src/backend/PortabilityService.Functions/Targets.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.Azure.WebJobs;
```

Comment:
Do we happen to have any tests for this? Or the functions in general?

Reply:
Tests are under \test\backend\PortabilityService.Functions.Tests for those Functions that are doing more work than returning a static string